pkg/openstack/loadbalancer_test.go

```diff
@@ -1543,3 +1543,99 @@ func Test_nodeAddressForLB(t *testing.T) {
 		})
 	}
 }
+
+func TestLbaasV2_getNetworkID(t *testing.T) {
+	lbaas := &LbaasV2{
+		LoadBalancer: LoadBalancer{
+			opts: LoadBalancerOpts{
+				LBClasses: map[string]*LBClass{
+					"lbclassKey": {
+						NetworkID: "lb-class-network-id-1234",
+					},
+				},
+				NetworkID: "default-lb-class-networkId",
+			},
+		},
+	}
+
+	type testArg struct {
+		service *corev1.Service
+	}
+	tests := []struct {
+		name    string
+		lbaas   LbaasV2
```

Comment:
I suggest using a pointer here for consistency with the function receiver. (or you can use a bare value if you want, but then please define the variable `lbaas` as a bare value too line 1548)

```suggestion
		lbaas   *LbaasV2
```